### ISSUE TITLE
TimeoutOverflowWarning when building the site

### ISSUE THREAD

**Issue by XhmikosR** [nodejs/nodejs.org#2442]
When building the site with `npm run build` I get:

```
TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
```

I'm on Node.js 12.8.1 on Windows 10 64-bit but it happens on Travis too. It's just not shown because currently the build script is inside tap. You can see it on my fork though https://travis-ci.org/XhmikosR/nodejs.org/builds/574910707#L216

IMO we should run `npm run build` in an npm script and not inside tap.

**Comment by XhmikosR** [comment]
I managed to trace this down to

```

C:\Users\xmr\Desktop\nodejs.org>set NODE_OPTIONS=--trace-warnings

C:\Users\xmr\Desktop\nodejs.org>npm start

> nodejs.org@ start C:\Users\xmr\Desktop\nodejs.org
> npm run serve


> nodejs.org@ serve C:\Users\xmr\Desktop\nodejs.org
> node server.js

[metalsmith] build/static started
[metalsmith] build/layouts started
Server running at http://localhost:8080/en/
(node:2584) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
    at getTimerDuration (internal/timers.js:366:13)
    at ClientRequest.setTimeout (_http_client.js:760:11)
    at Req._send (C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:138:29)
    at C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:50:21
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
(node:2584) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
    at getTimerDuration (internal/timers.js:366:13)
    at ClientRequest.setTimeout (_http_client.js:760:11)
    at Req._send (C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:138:29)
    at C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:50:21
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
(node:2584) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
    at getTimerDuration (internal/timers.js:366:13)
    at ClientRequest.setTimeout (_http_client.js:760:11)
    at Req._send (C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:138:29)
    at C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:50:21
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
(node:2584) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
    at getTimerDuration (internal/timers.js:366:13)
    at ClientRequest.setTimeout (_http_client.js:760:11)
    at Req._send (C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:138:29)
    at C:\Users\xmr\Desktop\nodejs.org\node_modules\hyperquest\index.js:50:21
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
[metalsmith] build/layouts finished: 674.879ms
[metalsmith] build/en started
[metalsmith] build/en finished: 6396.127ms
```

```
C:\Users\xmr\Desktop\nodejs.org>npm ls hyperquest
nodejs.org@ C:\Users\xmr\Desktop\nodejs.org
`-- node-version-data@1.0.1
  `-- jsonist@1.3.0
    `-- hyperquest@1.2.0
```

So, this needs node-version-data to bump jsonist. @rvagg any chance you could take care of this?

**Comment by rvagg** [comment]
Published node-version-data@1.1.0 which should address this. Thanks for all the diligence around this repo @XhmikosR, good work!
